telegram/utils/webhookhandler.py

```diff
@@ -80,6 +84,23 @@ def do_POST(self):
                              update.update_id)
             self.server.update_queue.put(update)
 
-        else:
-            self.send_error(403)
+        except _InvalidPost as e:
+            self.send_error(e.http_code)
             self.end_headers()
+
+    def _validate_post(self):
```

Comment:
your eyes are working properly :)
i forgot to call `self._validate_post()` instead of the `if` statement in `do_POST()`